### PR TITLE
[#3950] Make service catalogue more consistent with Signicat template

### PR DESCRIPTION
**Pull request by SilviaAmAm**
Fixes https://github.com/open-formulieren/open-forms/issues/3950 (first checkbox)

**Context**
Signicat, one of the Identity Providers for eHerkenning, said that the service catalogue that we generated was 'invalid'. The Signicat template can be found here: https://developer.signicat.com/broker/eherkenning-broker/eherkenning-service-catalogue.html#introduction

The noticed differences are mentioned in the ticket.

The service catalogue gets validated against this XSD: https://afsprakenstelsel.etoegang.nl/display/as/Service+catalog (under `XML schema Service Catalog`)

### CODE REVIEW

**Review by vaszig — APPROVED** [review]
Don't have a lot of experience in xml-xsd so I don't find something weird. 

I am not sure about the last bullet in the task "They don't have the signature node => The XSD has no value for the `minOccurs` and the default is `1` (https://www.w3.org/TR/xmlschema-0/#OccurrenceConstraints). So I think we need to leave this field in?"

I guess you are right.

**Comment by SilviaAmAm** [comment]
Discussion with Joeri:
- Re-add the sets for entity concernde types allowed

File: digid_eherkenning/migrations/0008_alter_eherkenningconfiguration_eh_requested_attributes_and_more.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Generated by Django 4.2.10 on 2024-03-07 16:49
```
Comment (vaszig) [inline digid_eherkenning/migrations/0008_alter_eherkenningconfiguration_eh_requested_attributes_and_more.py:1]: Do we have separate migrations on purpose?

File: digid_eherkenning/models/eherkenning.py [hunk 2]
```diff
@@ -9,6 +9,56 @@
 from .base import BaseConfiguration
 
 
+def get_default_requested_attributes_eherkenning():
+    return [
+        {
+            "name": "urn:etoegang:1.11:attribute-represented:CompanyName",
+            "required": True,
+            "purpose_statements": {
+                "en": "For testing purposes.",
```
Comment (vaszig) [inline digid_eherkenning/models/eherkenning.py:18]: Are these `purpose_statements` going to be always the same?This is from the example concerning eIDAS right?

File: docs/metadata.rst [hunk 3]
```diff
@@ -17,3 +17,31 @@ download it using cURL or similar tools).
...
+-------------------
+
+Configuring RequestedAttribute
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the field ``RequestedAttribute`` one can specify all the attributes that may be requested by the service
+when a company/person logs in with eHerkenning or eIDAS.
+
+The values specified need to come from the "`Attribuutcatalogus <https://afsprakenstelsel.etoegang.nl/display/as/Attribuutcatalogus>`_"
+(there are a 'generiek', 'natuurlijke personen' and 'non-natuurlijke personen' catalogue).
```
Comment (vaszig) [inline docs/metadata.rst:31]: I am not sure but should it be like this?

```suggestion
(there is a 'generiek', 'natuurlijke personen' and 'non-natuurlijke personen' catalogue).
```

File: digid_eherkenning/models/eherkenning.py (hunk 2 above)
Comment (SilviaAmAm) [reply digid_eherkenning/models/eherkenning.py:18]: All these fields are configurable, but this is just for the default!

**Review by vaszig — APPROVED** [review]

**Pull request merged**